config.json

```diff
@@ -8,7 +8,7 @@
         "name": "opa-args",
         "description": "Arguments for opa-docker-authz",
         "settable": ["value"],
-        "value": ["-policy-file", "policy.rego"]
+        "value": ["-policy-file", "policy.rego", "-quiet"]
```

Comment:
Can we skip this change or is it difficult to configure the `-quiet` flag otherwise?

Reply:
I'm not sure. I haven't tried deploying the plugin without that configuration. It seemed necessary, but I can try removing it and see if that works.

Reply:
I'm just thinking that for the default use case of someone trying out the plugin it's nice to see the requests logged. If you want to turn off the request logging, you can do that, but for the default/kick the tires behavior, it's useful.

Reply:
Oh! My bad. I misunderstood the purpose of the `config.json` file. It works flawlessly without this change.